setup.py

```diff
@@ -11,7 +11,9 @@
 
 setup_args = {}
 install_requires = ['param>=1.9.3,<2.0', 'numpy>=1.0', 'pyviz_comms>=0.7.3',
-                    'panel>=0.7.0', 'pandas']
+                    "panel>=0.7.0,<0.9.0; python_version<'3.6.0'",
+                    "panel>=0.7.0; python_version>='3.6.0'",
```

Comment:
Shouldn't it be?
```suggestion
                    "panel>=0.9.0; python_version>='3.6.0'",
```

Reply:
No, because holoviews still only requires a minimum panel version of 0.7.0. We're only adding a maximum panel version to the requirements (and it only applies on Python 3.5).